cypress/e2e/ResumeParserAPI/apiInfo.cy.js

```diff
@@ -0,0 +1,211 @@
+
+describe('Resume Parser API Info page', () => {
+    before(() => {
+        cy.visit(RESUME_PARSER_API);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        // Check if the "Accept Cookies" button is visible or exists
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to click "Api Info" tab', () => {
+        cy.get('#details-tab > .d-md-flex > span').click();
+        cy.componentVisiblityCheck('#details');
+    });
+
+    it('3. should display the description', () => {
+        cy.get('article p').first().should('contain.text', 'Resume parsing API is a hosted service that takes a resume as an input that can be in PDF or MS Word format');
+    });
+
+    it('4. should display the use cases', () => {
+        const useCases = [
+          'Search, Sort and Store resumes',
+          'Enhance your talent database.',
+          'Save time and efforts.',
+          'Eliminate manual intervention. Eliminate privacy issues.',
+          'Faster closing of open positions'
+        ];
+        useCases.forEach((useCase) => {
+          cy.get('ul').should('contain.text', useCase);
+        });
+    });
+
+    it('5. should mention getting an API key for parsing a CV/resume', () => {
+        cy.get('h4').contains('Parsing a CV/resume on a URL').next('p').should('contain.text', 'get your API key');
+    });
+
+    it('6. Test to check "Beer Slider" is present', () => {
+        cy.componentVisiblityCheck('#beer-slider');
+    });
+
+    it('4. Test to check "After Image" is visible', () => {
+        cy.componentVisiblityCheck('img[alt="Resume in JSON format"]');
+    });
+
+    it('7. Test to check Slider visibility after resizing', () => {
+        // Change the width style dynamically and check visibility
+        cy.get('.beer-reveal').invoke('css', 'width', '75%');
+        cy.componentVisiblityCheck('.beer-handle');
+    });
+
+    it('8. Test to check Beer Slider visibility when hidden', () => {
+        cy.get('#beer-slider').invoke('css', 'display', 'none');
+        cy.get('#beer-slider').should('not.be.visible');
+    });
+    
+    it('9. Test to check the shell code snippet is present', () => {
+        // Check if the <pre> element with class "language-shell" is present
+        cy.get('pre.language-shell').should('exist');
+    });  
+      
+    it('10. Test to check the JavaScript code snippet is present', () => {
+        // Check if the <pre> element with class "language-javascript" is present
+        cy.get('pre.language-javascript').should('exist');
+    });
+
+    it('11. Test to check the shell command snippet is present', () => {
+        // Check if the <pre> element with class "language-shell" is present
+        cy.get('pre.language-shell').should('exist');
+    });  
+    
+    it("12. Test to check visibility of 'Bulk processing of resumes' heading", () => {
+        // Check if the <h4> heading is present
+        cy.get('article > :nth-child(20)').should('exist');
```

Comment:
seeing nth child can you find alternative

Reply:
i didnt got any alternative s i am getting class  from cypress and using it